webapp/client/src/components/FormNavButtons.stories.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import FormNavButtons from "./FormNavButtons";
+
+export default {
+  title: "Forms/Nav Buttons",
+  component: FormNavButtons,
+  decorators: [
+    (
+      Story // Render context from app/templates/basis/base.html
+    ) => (
+      <div className="main-content">
+        <div className="mt-4">
+          <div className="col-lg-9 col-md-10 col-xs-12 p-0">
+            <Story />
+          </div>
+        </div>
+      </div>
```

Comment:
I don't think I get this part. Is this just copied from the base.html? So *should* we want to change that, we'd need to change it here, too?

Reply:
Yeah, this is a crutch for now to get components to look the same in storyboard as they would in the app. I'm not sure we need it at all in the end. I've made a note to revisit it at a later point.